spring-cloud-vault-core/src/main/java/org/springframework/cloud/vault/VaultProperties.java

```diff
@@ -58,6 +58,12 @@
 	private int connectionTimeout = 5000;
 
 	/**
+	 * Used to force a PropertySourceLocator ordering.
+	 * This is useful to use Vault as an override on consul properties;
+	 */
+	private int propertySourceLocatorPriority = 0;
```

Comment:
That will make the property `spring.cloud.vault.config.order` rather than `spring.cloud.vault. property-source-locator-priority`.